test/spec/validations/ambiguous_crossing_tags.js

```diff
@@ -159,7 +174,7 @@ describe('validationAmbiguousCrossingTags', () => {
       {}
```

Comment:
Should this empty parentheses contain any text, like `'crossing:markings': 'no'`?

Reply:
Very good eye! 
This is an attempt to test for candidate crossings like this: 

![image](https://github.com/facebook/Rapid/assets/1887955/44e8c1d0-613b-4127-94ed-1eec171eeb99)


'crossing candidate' nodes are usually not tagged with anything at all- they're just nodes with two parents, one of which is a crossing way, but other than that they _often_ have no tags whatsoever.

Reply:
I have updated the test name to be clearer that this is a test of the crossing candidate nodes